spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  before(:all) do
+    @user = User.create(name: 'Cook', email: 'cooker@masterchef.com', password: '123456', confirmed_at: Time.now)
+    @recipe = Recipe.create(user: @user, name: 'recipe 1', description: 'description', public: true)
```

Comment:
- [ ] two of your tests are failing kindly fix that 
![image](https://github.com/c00p75/recipe-app/assets/66097371/42289204-8d64-4a47-987e-245fa0b3dc0f)

Reply:
Ok, we will. Thanks